### PR TITLE
Update values

### PR DESCRIPTION
**Pull request by paddatrapper**

### CODE REVIEW

File: build/ci/production-values.yaml [hunk 1]
```diff
@@ -1,20 +1,22 @@
 sandbox: "false"
 contentSandbox: "false"
-deployEnv: production  
+deployEnv: production
 previousServicesCount: "3"
 restore:
-  enabled: false
+  enabled: true
```
Comment (BrendanGalloway) [inline build/ci/production-values.yaml:6]: need a reference to the specific tarball here

File: build/ci/production-values.yaml [hunk 2]
```diff
@@ -85,51 +80,33 @@ varnish:
     between_bytes_timeout: 60  # Default 60s
   secrets:
     varnish:
-      ref: varnish-secret-production
+      ref: varnish-secret
```
Comment (BrendanGalloway) [inline build/ci/production-values.yaml]: have we created this? IIRC it just needs to be a seed?

File: build/ci/production-values.yaml [hunk 3]
```diff
@@ -85,51 +80,33 @@ varnish:
...
       path: /*
       serviceName: nginx
       port: 80
   secrets:
-    originTls: 
-      ref: origin-tls-prod2
-    originIlTls:
-      ref: origin-il-tls-prod2 
+    originTls:
+      ref: origin-tls
```
Comment (BrendanGalloway) [inline build/ci/production-values.yaml:104]: Did we pin down whether this should be origin-tls or origin-tls-prod2?  I think the sealed secret in infra is prod2?

File: build/ci/production-values.yaml [hunk 4]
```diff
@@ -151,30 +128,28 @@ monitor:
...
-  FRONT_END_URL:  "http://www.sefaria.org" 
+  FRONT_END_URL:  "http://contextus.org"
   OFFLINE: false
   DOWN_FOR_MAINTENANCE: false
-  MAINTENANCE_MESSAGE: "Sefaria is down for scheduled maintenance. We expect to be back online within 15 minutes. Thank you for your patience."
+  MAINTENANCE_MESSAGE: "Contextus is down for scheduled maintenance. We expect to be back online within 15 minutes. Thank you for your patience."
   GLOBAL_WARNING: false
-  GLOBAL_WARNING_MESSAGE: "Sefaria will be in <b>Read-Only</b> mode for scheduled maintenance from 4:45pm-6:45pm Pacific time. Edits will <b>not</b> be saved during that time."
+  GLOBAL_WARNING_MESSAGE: "Contextus will be in <b>Read-Only</b> mode for scheduled maintenance from 4:45pm-6:45pm Pacific time. Edits will <b>not</b> be saved during that time."
   SITE_PACKAGE: "sites.sefaria"
```
Comment (BrendanGalloway) [inline build/ci/production-values.yaml]: I think this needs to be sites.s4d @stevekaplan123

**Review by BrendanGalloway — CHANGES_REQUESTED** [review]

File: build/ci/production-values.yaml (hunk 2 above)
Comment (paddatrapper) [reply build/ci/production-values.yaml]: It's not in the repo, so think it still needs creating

File: build/ci/production-values.yaml (hunk 3 above)
Comment (paddatrapper) [reply build/ci/production-values.yaml:104]: The secret is origin-tls in cloudflare-cert.yaml

**Pull request merged**